flink-runtime-web/web-dashboard/src/app/services/task-manager.service.ts

```diff
@@ -50,25 +50,38 @@ export class TaskManagerService {
   }
 
   /**
-   * Load TM logs
+   * Load TM log list
    * @param taskManagerId
    */
-  loadLogs(taskManagerId: string) {
-    return this.httpClient.get(`${BASE_URL}/taskmanagers/${taskManagerId}/log`, {
-      responseType: 'text',
-      headers: new HttpHeaders().append('Cache-Control', 'no-cache')
-    });
+  loadLogList(taskManagerId: string) {
+    return this.httpClient
+      .get<{ logs: Array<{ name: string; size: number }> }>(`${BASE_URL}/taskmanagers/${taskManagerId}/logs`)
```

Comment:
extract to a TS interface.
```suggestion
      .get<{ logs: TaskLog[] }>(`${BASE_URL}/taskmanagers/${taskManagerId}/logs`)
```